protex/tests/test_simulation.py

```diff
@@ -18,7 +18,7 @@
     reason="Skipping tests that cannot pass in github actions",
 )
 def test_outline():
-    from simtk.openmm.app import DCDReporter, StateDataReporter
+    from openmm.app import DCDReporter, StateDataReporter
```

Comment:
We break compatibility with OpenMM Versions <7.6 if we import "from openmm" and not "from simtk.openmm", right?

Reply:
yes! I wrapped this in a `try except` block that falls back on stimk